config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   root to: 'manifests#index'
-  resources :manifests, only: [:new, :create]
+  resources :manifests, only: [:new, :show]
```

Comment:
we need to add back in `create` here so the specs don't fail :) (should be `[:new, :create, :show]`)